code/modules/vehicles/scooter.dm

```diff
@@ -82,25 +82,38 @@
 		set_density(FALSE)
 	return ..()
 
-/obj/vehicle/ridden/scooter/skateboard/Bump(atom/A)
+/obj/vehicle/ridden/scooter/skateboard/Bump(atom/bumped_thing)
 	. = ..()
-	if(!A.density || !has_buckled_mobs() || world.time < next_crash)
+	if(!bumped_thing.density || !has_buckled_mobs() || world.time < next_crash)
 		return
 
 	next_crash = world.time + 10
 	var/mob/living/rider = buckled_mobs[1]
 	rider.adjustStaminaLoss(instability*6)
 	playsound(src, 'sound/effects/bang.ogg', 40, TRUE)
-	if(!iscarbon(rider) || rider.getStaminaLoss() >= 100 || grinding)
+	if(!iscarbon(rider) || rider.getStaminaLoss() >= 100 || grinding || iscarbon(bumped_thing))
 		var/atom/throw_target = get_edge_target_turf(rider, pick(GLOB.cardinals))
 		unbuckle_mob(rider)
+		if((istype(bumped_thing, /obj/machinery/disposal/bin)))
+			rider.Paralyze(8 SECONDS)
+			rider.forceMove(bumped_thing)
+			src.forceMove(bumped_thing)
+			visible_message(span_danger("[src] crashes into [bumped_thing], and gets dumped straight into it!"))
+			return
 		rider.throw_at(throw_target, 3, 2)
 		var/head_slot = rider.get_item_by_slot(ITEM_SLOT_HEAD)
 		if(!head_slot || !(istype(head_slot,/obj/item/clothing/head/helmet) || istype(head_slot,/obj/item/clothing/head/hardhat)))
 			rider.adjustOrganLoss(ORGAN_SLOT_BRAIN, 5)
 			rider.updatehealth()
-		visible_message(span_danger("[src] crashes into [A], sending [rider] flying!"))
-		rider.Paralyze(80)
+		visible_message(span_danger("[src] crashes into [bumped_thing], sending [rider] flying!"))
+		rider.Paralyze(8 SECONDS)
+		if(iscarbon(bumped_thing))
+			var/mob/living/carbon/victim = bumped_thing
+			var/grinding_mulitipler = 1
+			if(grinding)
+				grinding_mulitipler = 2
+			victim.Knockdown(4 * grinding_mulitipler SECONDS)
+			victim.Paralyze(1 * grinding_mulitipler SECONDS)
```

Comment:
Let's go slow on the balance changes, please.

Reply:
These are less than half of what the skater takes when they collide. This very much improves the feel of skateboarding and opens combat opportunities in which one person skates into someone exposing them to attack from a partner. Seems relatively balanced, and it's not going to shake the meta.
I'd be open to your proposed changes.

Reply:
If this PR doesn't change the effectiveness of skateboards as a potential weapon of combat then this PR has failed. I'd be more than willing to accept a balance tag.

Reply:
The knockdown should be enough for those purposes. No need to `Paralyze()`.

Reply:
I've given it some thought and this is a good solution. It will be done.